### PR TITLE
#1271 Reply with estimation changed value.

### PR DESCRIPTION
**Pull request by criske**
- Unignored test
- Updated Step api to be able to receive data from the previous Step in chain.

Pr for #1271

### CODE REVIEW

**Comment by zoeself** [comment]
@criske thank you for your Pull Request. I'll assign someone to review it soon. 

 If this PR solves a ``todo`` from the code, please don't forget to remove it.

**Comment by criske** [comment]
@amihaiemil I think with this aproach we can send the new estimation from `IssueEstimationChanged` to 'UpdateTaskEstimation`,
since it's next one in chain.

**Comment by zoeself** [comment]
@amihaiemil I couldn't find any assignee for this task. This is either because there are no contributors with role ``REV`` available or because the project does not have enough funds.

Please, make sure there is at least one available contributor with the required role and the project can afford to pay them.

**Comment by amihaiemil** [comment]
@criske scuze, am uitat de API-u Event si faptu ca nu prea avem acces usor la vechea estimare in contextul de fata. 

As renunta la SendReply in cazul asta si as folosi functie anonima, ca in alte cazuri, de exemplu [aici](https://github.com/self-xdsd/self-core/blob/master/self-core-impl/src/main/java/com/selfxdsd/core/managers/StoredProjectManager.java#L285). Sincer nici nu mai stiu acum la ce e folosit SendReply (vad ca in manager nu e folosit nicaieri) :D 

Iar textul sa fie doar: "Estimation changed to X min at Timestamp" (poti lua noua estimare din event.issue().estimation()) Si pt text efectiv foloseste project.language(), sa fie traductibil : D

Altfel treaba asta cu derive, nu prea vad la ce am folosi-o. Poate ramane ca PoC pt viitor : D

**Comment by amihaiemil** [comment]
@criske a, SendReply cred ca e folosit doar cand raspunde la un Comment, sa appenduiasca acolo si Comment-ul anterior. 

Legat de asta cu derive(...) - nu cred ca e o idee buna, desi la prima vedere mi se pare sexy. Ideea ar fi ca un Step sa lucreze doar cu Event-ul, de acolo sa isi ia datele cu care lucreaza, exclusiv :D

**Comment by criske** [comment]
@amihaiemil ah ok, pai daca e nevoie doar de noua estimare comment atunci se simplifica lucrurile... ii dau revert si o sa folosesc un lambda in loc de `SendReply`.

File: self-core-impl/src/test/java/com/selfxdsd/core/managers/StoredProjectManagerTestCase.java [hunk 1]
```diff
@@ -1871,6 +1883,12 @@ public void handlesIssueLabelsChangesOfEstimation() {
         manager.issueLabelsChanged(event);
 
         Mockito.verify(task, Mockito.times(1)).updateEstimation(30);
+        final ArgumentCaptor<String> commentCaptor = ArgumentCaptor
+            .forClass(String.class);
+        Mockito.verify(comments, Mockito.times(1))
+            .post(commentCaptor.capture());
+        MatcherAssert.assertThat(commentCaptor.getValue(), Matchers
+            .startsWith("Estimation changed to ``30`` minutes at ``20"));
```
Comment (criske) [inline self-core-impl/src/test/java/com/selfxdsd/core/managers/StoredProjectManagerTestCase.java:1891]: //TODO we'll need to fix this test in 2100 :D

Comment (amihaiemil) [reply self-core-impl/src/test/java/com/selfxdsd/core/managers/StoredProjectManagerTestCase.java:1891]: took me a while to understand haha

**Pull request merged**